npm/webpack-preprocessor/README.md

```diff
@@ -44,9 +44,9 @@ module.exports = (on) => {
 
 ## Examples
 
-- [React app](examples/react-app) shows how to point Cypress at Webpack configuration from `react-scripts` dependency
-- [use-babelrc](examples/use-babelrc) shows how to use your project's `.babelrc` with Webpack
-- [use-ts-loader](examples/use-ts-loader) shows how to transpile TypeScript specs following [Webpack TypeScript guide](https://webpack.js.org/guides/typescript/)
+- [React app](https://github.com/cypress-io/cypress/tree/develop/npm/webpack-preprocessor/examples/react-app) shows how to point Cypress at Webpack configuration from `react-scripts` dependency
```

Comment:
Do we want our links to point to develop (pending release), rather than master (currently released)?